actions/store_message_info.js

```diff
@@ -293,27 +293,26 @@ module.exports = {
 				result = msg.webhookID;
 				break;
 			case 22:
-				const embed = msg.embeds[0];
-				delete embed.message;
-				let Embeds = [];
-				JSON.stringify(embed, function(key, value) {
-					if (typeof value === 'object' && value !== null) {
-						if (Embeds.indexOf(value) !== -1) {
-							return;
+				if (msg.embeds.length != 0) {
+					const embed = msg.embeds[0];
+					delete embed.message;
+					if (embed) {
```

Comment:
redundant if statement. If embed didn't exist it would've error on the delete function above this line.

Reply:
Lol, my bad